src/style.css

```diff
@@ -0,0 +1,168 @@
+* {
+  box-sizing: border-box;
+
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+body,
+h2,
+h3 {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background: var(--background-color);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  --bottom-color: #00acdc;
+  --text-color: #060b35;
+  --grey-text: #616070;
+  --background-color: #fff;
+}
+
+.foto_image {
+  width: 160px;
+  height: 134px;
+  margin-left: 19px;
+  margin-right: 19px;
+  margin-top: 32px;
+}
+
+.title {
+  width: 166px;
+  height: 36px;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: var(--text-color);
+  margin-left: 16px;
+  margin-right: 16px;
+  margin-top: 40px;
+}
+
+.product-code {
+  display: flex;
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  width: 98px;
+  height: 14px;
+  color: var(--grey-text);
+  margin-left: 16px;
+  margin-top: 4px;
+  margin-right: 84px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: flex;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
+  list-style: none;
+  margin-top: 16px;
+  margin-left: 4px;
+}
+
+.stars__star:first-child {
+  margin-left: 16px;
+}
+
+.stars .stars__star:nth-of-type(-n+4) {
```

Comment:
You added `active` state only for `4-stars` case. But in order to meet the requirements you should create a general solution:
![image](https://user-images.githubusercontent.com/25587299/236860885-9037eaf5-b767-444e-bc2f-8f4ebea68a7f.png)

Reply:
I don`t understand what I need to do?